backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java

```diff
@@ -365,4 +418,362 @@ public FutureExperimentRunDAO(Executor executor, FutureJdbi jdbi, UAC uac) {
         .thenCompose(unused -> artifactHandler.deleteArtifacts(runId, optionalKeys), executor)
         .thenCompose(unused -> updateModifiedTimestamp(runId, now), executor);
   }
+
+  private InternalFuture<UserInfo> getCurrentLoginUserInfo() {
+    return FutureGrpc.ClientRequest(
+        uac.getUACService().getCurrentUser(Empty.newBuilder().build()), executor);
+  }
```

Comment:
Remove

https://github.com/VertaAI/docs/blob/main/engineering-practices/api-implementation/performance.md#avoid-layers-of-abstraction

Reply:
Removed